src/components/Layout/Footer/index.js

```diff
@@ -5,12 +5,13 @@ import { images } from '../../../constants/images';
 import './style.scss';
 
 const Footer = () => {
+  const year = new Date().getFullYear();
   return (
     <footer className="footer">
       <div className="container">
         <img className="logo" src={images.logo_footer} alt="logo" />
         <div>
-          <p className="m-0">Ⓒ 2018 GXC World Pte Ltd</p>
+          <p className="m-0">Ⓒ { year } GXC World Pte Ltd</p>
```

Comment:
Spaces inside curly braces not recommened in react.

See airbnb's react style guide - https://github.com/airbnb/javascript/tree/master/react#spacing